integrations/groq/integration.definition.ts

```diff
@@ -1,33 +1,13 @@
-import { llm } from '@botpress/common'
-import { IntegrationDefinition, z } from '@botpress/sdk'
-
-const model = z
-  .enum(['llama3-8b-8192', 'llama3-70b-8192', 'mixtral-8x7b-32768', 'gemma-7b-it'])
-  .describe('Model to use for content generation')
-  .default('mixtral-8x7b-32768')
-
-export type Model = z.infer<typeof model>
+import { IntegrationDefinition, interfaces } from '@botpress/sdk'
 
 export default new IntegrationDefinition({
   name: 'groq',
   version: '0.0.1',
   readme: 'hub.md',
   icon: 'icon.svg',
-  actions: {
-    generateContent: {
-      title: 'Generate Content',
-      description: 'Generate content using any LLM supported by Groq',
-      input: {
-        schema: llm.schemas.GenerateContentInputSchema.extend({ model }),
```

Comment:
**issue:** This is breaking. It is unfortunately not possible to do such a thing with interfaces, unleash we define an entity.

Reply:
the more I look into it, the more I think it could be done with an entity